tests/e2e/rudder-cdn-js/README.md

```diff
@@ -0,0 +1,26 @@
+# Getting Started with Rudder Typer
+
+## Prerequisite(If you want to use your tracking plan)
+
+1. You have setup the tracking plan from the dashboard.
+2. Generated PAT from dashboard settings.
+3. Did the authentication step using PAT and email to generate `ruddertyper.yml` and `plan.json` file in the root level of the repo.
+4. Move ruddertyper.yml file from root level to tests/e2e/rudder-npm-js this folder.
+5. Replace the plan.json in rudder-npm-js/src/analytics with the one you generated.
+6. Delete `index.ts` and `rudder.ts` inside analytics folder.
+7. Run the command `npx rudder-typer@next`
```

Comment:
```suggestion
7. Run the command `npx rudder-typer`
```
we can remove @next everywhere as in the latest PR we have removed this tag

Reply:
Without the next tag, it is downloading an older version like beta.5.